dependencies/windows/install-boost/install-boost.R

```diff
@@ -122,7 +144,7 @@ b2_build_args <- function(bitness) {
       "--abbreviate-paths",
       sprintf("variant=%s", variant),
       sprintf("link=%s", link),
-      sprintf("runtime-link=shared", link),
```

Comment:
The formatting argument was being ignored here, so in fact we were always using shared linking to the runtime. Because we do ship the required runtime libraries (necessary for Electron) we don't need to perform static runtime linking here.